ui-lib/src/components/Tag/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import { border, BorderProps, space, SpaceProps, color, ColorProps } from "styled-system";
+import Text from "@components/Text";
+
+export type Props = React.PropsWithChildren<{
+  /**
+   * Changes the appearance based on the active state.
+   */
+  active?: boolean;
+  /**
+   * Tag style.
+   */
+  type?: "plain" | "translucent" | "outlined";
+}>;
+
+function getColor({ type, active }: Props) {
+  switch (type) {
+    case "translucent":
+    case "outlined":
+      return "palette.v2.primary.Dark";
+    default:
+      return active ? "palette.v2.background.default" : "palette.v2.primary.Dark";
```

Comment:
Why is this color uppercase? 🤔

Reply:
The key is currently uppercased in the palette in the develop branch : https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/ui-lib/src/styles/palettes/light.json#L27

@jgiamporcaro-ledger made a fix but it's still not merged yet I think ❓

Reply:
Yeah, currently pending. I'm going to isolate the fix by cherry-pinking it from my branch and send a fix PR. Please review it asap and we'll be able to merge it.